sympy/utilities/autowrap.py

```diff
@@ -133,7 +133,9 @@ def _generate_code(self, main_routine, routines):
             routines, self.filename, True, self.include_header,
             self.include_empty)
 
-    def wrap_code(self, routine, helpers=[]):
+    def wrap_code(self, routine, helpers=None):
```

Comment:
Why doesn't LGTM like this?

Reply:
Apparently `helpers` is initialized once, so any additions will be there for a later call without providing the optional argument.

There is a good example of that here:
https://github.com/sympy/sympy/blob/92c18446ca8f721f1106e0a8c076fc538732cb9b/release/fabfile.py#L200-L214

As noted line 213 may seem useless, since the method returns another value on the next line, but it is the whole trick of it (and why I didn't remove this particular instance).

Reply:
That's a useful thing to keep in mind. Thanks for the explanation.